tailwind.config.js

```diff
@@ -11,10 +11,11 @@ module.exports = {
 
   purge: [
     "./app/views/**/*.html.erb",
-    "./app/components/**/*.rb",
```

Comment:
Can you explain the changes to this file, please?

Reply:
Yup! Rather than having
```
purge: [
  "./app/components/**/*html.erb",
  "./app/components/**/*.rb",
  "./app/components/**/*.html",
```

You can specify all extensions with `{extensions}

So the above just becomes
```
purge: [
  "./app/components/**/*.{html.erb,rb,html}",
]
```

I personally prefer the single line approach as it's easier to lump concepts together, but I'm completely happy separating it back out

Reply:
Ahh, got it. That makes a ton of sense. Thanks for the explanation!